portfolio/src/main/java/com/google/sps/servlets/UtilityFunctions.java

```diff
@@ -52,6 +52,9 @@ public static void addToDatastore(String name, String email, long dateTime, Stri
           .set("comment", comment)
           .set("parentid", parentId)
           .set("rootid", rootId)
+          .set("upvotes", upvotes)
+          .set("downvotes", downvotes)
+          .set("score", upvotes-downvotes)
```

Comment:
Seems unnecessary to save "score" in the datastore since it can be derived from upvotes and downvotes.

If you want to use "score" as an index, I guess you can skip downvotes?